.gitignore

```diff
@@ -117,3 +117,4 @@ temp
 .vscode
 /test/Tryouts/Databases/*
 bench/Micro.Benchmark/BenchmarkDotNet.Artifacts/
+/src/Raven.Server/*.DotSettings
```

Comment:
how you ended up with *.DotSettings file inside that dir? We have it in the root

Reply:
Not sure. I think it was added by R# somehow.